### ISSUE TITLE
Requesting enhancement exception request for - allow users to recover from expansion failure

### ISSUE THREAD

**Issue by gnufied** [kubernetes/enhancements#1529]
# Enhancement Description
- Allow users to recover from volume expansion failure
- Kubernetes Enhancement Proposal: https://github.com/kubernetes/enhancements/pull/1516
- Primary contact (assignee): @gnufied 
- Responsible SIGs: sig-storage, sig-api-machinery
- Enhancement target (which target equals to which milestone):
  - Alpha release target (x.y)
  - Beta release target (x.y) : 1.18
  - Stable release target (x.y): 1.19
- Additional time needed: 5 days (02/05/2020)
- Reason this enhancement is critical for this milestone: If we can't get this done in milestone 1.18 - it would push back volume expansion going GA for 2-3 more releases, which is really a long time a feature has been in beta.
- Risks from adding code late: In general I think implementation itself should not require exception, just the enhancement requires exception. Also - the change should be self-contained to not affect other areas of k8s.
- Risks from cutting enhancement: As I mentioned above volume expansion feature has been in beta for awhile and we need to move it to GA.  Risk from not having this enhancement in 1.18 is - the whole of volume expansion feature has to be pushed back by 2 or more releases.

Volume expansion has been in beta for awhile and is being used by many users in production, but one of the problems that exists in current workflow is, if expansion fails - there is no easy way for an user to recover from failure herself. This does not make PVC unusable but it requires admin intervention(as detailed in the linked KEP).

sig-storage believes that before we can move volume expansion feature as a whole to GA, this problem should be fixed and linked enhancement addresses that. 

/sig storage

cc @saad-ali @msau42 @jsafrane @bswartz @davidz627

**Comment by gnufied** [comment]
Closing this in favour of email discussion.
